compiler/transforms/snax_copy_to_dma.py

```diff
@@ -192,12 +192,12 @@ def match_and_rewrite(self, op: CopyOp, rewriter: PatternRewriter):
         if tsl_source.data.offset:
             offset = Constant.from_int_and_width(tsl_source.data.offset, IndexType())
             pointer_src = Addi(pointer_src, offset, IndexType())
-            ops_to_insert.append(offset)
+            ops_to_insert.extend([offset, pointer_src])
 
         if tsl_dest.data.offset:
             offset = Constant.from_int_and_width(tsl_dest.data.offset, IndexType())
             pointer_dst = Addi(pointer_dst, offset, IndexType())
-            ops_to_insert.append(offset)
+            ops_to_insert.extend([offset, pointer_dst])
```

Comment:
I'm assuming this was previously forgotten?

Reply:
![image](https://github.com/KULeuven-MICAS/snax-mlir/assets/47864363/c346e0b7-62d9-4a5f-9124-0feb87c21cbe)

Reply:
Haha allright